src/UI/templates/default/Listing/workflow.less

```diff
@@ -0,0 +1,175 @@
+@workflow-icon-width: 20px;
```

Comment:
Announce less variables in the description. See [here](https://github.com/ILIAS-eLearning/ILIAS/blob/release_5-3/src/UI/docu/rules.md#introduction-of-new-ui-components)